draft-ietf-quic-version-negotiation.md

```diff
@@ -387,6 +387,41 @@ lifetime of the connection; endpoints MUST NOT change the version that they
 consider to be in use based on the Version field of long headers as that field
 could be forged by attackers.
 
+# Minimum support
+
+Hosts can support this standard incrementally to simplify implementation, if
+not all features of the design are desired.
+
+{{INV}} and {{QUIC}} already describe how servers send Version Negotiation
+packets in response to long header packets with unsupported version fields, even
+if they support only one version. Doing so prevents long timeouts at the client,
+but servers might not send the packet multiple times for the same connection, or
+when under stress.
+
+If a server supports two or more QUIC versions and sends Version Negotiation
+packets, it MUST follow the requirements in {{vers-info}} and {downgrade}}.
+Failure to comply would leave the connection vulnerable to downgrade attacks, or
+cause connection failure, if the client initiates Incompatible Version
+Negotiation ({{incompat-vn}}).
+
+A server that supports two or more compatible QUIC versions MAY implement
+Compatible version negotiation ({{compat-vn}}) if it prefers one version over
+another.
+
+A client that supports two or more versions, and chooses to respond to a Version
+Negotiation packet with a connection attempt using one of the advertised
+versions, MUST include a Version Information Transport Parameter and process the
+server's transport parameter in accordance with {{downgrade}}.
+
+If a client sends a Version Information Transport Parameter in a packet using a
+version that is compatible with a version advertised in the transport parameter,
+it MUST support compatible version negotiation.
```

Comment:
Isn't this stuff required elsewhere.

Reply:
If so, I can't find it, but if one reads and fully understands the doc it's an unavoidable conclusion.I thought it would be useful to make it explicit

Reply:
OK, I guess for now it can go here, but I think generally it should go elsewhere.